src/github_refresh.py

```diff
@@ -78,27 +82,46 @@ def get_epv_list():
             if bool(dep_list):
                 for k in dep_list:
                     name = k.split(" ")[0]
-                    epv_list.append({
-                        "ecosystem": eco,
-                        "name": name
-                    })
+                    epv_list[eco].append(name)
             else:
                 logger.info("No deps found in the report for the ecosystem {}".format(eco))
+
+    logger.info("Fetching venus V2 stack report for the date {}".format(yest_date))
+    v2_stack_json = retrieve_dict("v2/daily/" + yest_date + ".json")
+    if v2_stack_json:
```

Comment:
Common code across v1 and v2 , can be moved into a common method.

Reply:
yes i know.. but very soon i will be removing v1 all together as there wont be any data coming in v1. let me know if you still want me to write the common stuff

Reply:
i kept it simple so that deletion of code is easy, in some days from now

Reply:
My suggestion is to create a common method. I do not know when we will get time to clean up V1 code.